docs/hvordan-utvikle-maler.adoc

```diff
@@ -0,0 +1,550 @@
+:docinfo: shared
+:source-highlighter: highlight.js
+:toc:
+
+= Utvikling av brevmaler i brevbakeren
+
+Brevbakeren inneholder brevmaler i form av vår egen Kotlin DSL(Domain specific language)
+DSL'en er utviklet for å kun inneholde de elementene vi trenger for å representere innholdet i brev i nav.
+
+Vi har med vilje utelatt enkelt funksjonalitet som ganging og deling ettersom vi mener at forretningslogikk bør holdes utenfor brevmaler.
+
+Mal-språket er laget spesielt for å ikke klare å lage tekniske feil i løsningen ved bestilling av ett brev, og at tekniske feil ikke fører til feil i innholdet.
```

Comment:
Forslag: Mal-spåket er laget spesifikt for å gjøre det mindre sannsynlig at man implementerer tekniske feil i maler og at disse fører til at et brev produserers med uheldige feil og mangler. Dette inkluderer blant null-safety, slik at det ikke er mulig å bruke nullable inndata på en måte hvor man risikerer NPE eller at teksten "null" havner i produserte brev.

Reply:
Push gjerne forslagene dine :)